build_r.R

```diff
@@ -5,6 +5,22 @@
 # Sys.setenv("CXX" = "/usr/local/bin/g++-8")
 # Sys.setenv("CC" = "/usr/local/bin/gcc-8")
 
+# Identify R-packages required for build_r.R to complete
```

Comment:
Thanks @zkurtz , and apologies for the inconvenience.

But this is excessive complexity. Can you just add the following to the top of the script:

```
library(testthat)
library(roxygen2)
library(devtools)
```

Then anyone who doesn't have those will have this script immediately fail, and can just install them with `install.packages`

Reply:
also please keep an eye on #629 ! We are trying to get CRAN-ready as fast as possible so hopefully you soon won't need to build from source.